concord-server/server/src/main/java/com/rabidgremlin/concord/resources/PhrasesResource.java

```diff
@@ -127,6 +126,19 @@ public Response getNextPhraseToLabel(@ApiParam(hidden = true) @Auth Caller calle
     }
   }
 
+  @POST
+  @Path("bulk")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Timed
+  public Response addPhrases(@ApiParam(hidden = true) @Auth Caller caller, UnlabelledPhrases phrases)
+  {
+    log.info("{} uploading json of phrases {}", caller, phrases);
```

Comment:
Does UnlabelledPhrases override toString() method? If not the logging here will be less useful

Reply:
It does, but it dumps all the phrases, better to log the size of the set.